modules/ROOT/pages/downgrade-runtime-fabric.adoc

```diff
@@ -0,0 +1,32 @@
+= Downgrading Runtime Fabric Versions
+
+If you need to downgrade Runtime Fabric version 2.6.x, you must:
+
+. Patch Runtime Fabric to the lower version
```

Comment:
1. Downgrade runtime fabric.
2. Perform additional steps to switch to daemon set log forwarding. 
(These steps are applicable to customers that use AM log forwarding, other customers do not need these steps).